app/views/entities/new.html.erb

```diff
@@ -11,7 +11,12 @@
     <%= form.label :amount %>
     <%= form.text_field :amount, class: 'form-control', placeholder: 12, type: "number", step: ".01" %>
   </div>
-  
+
+    <div class="mb-3">
```

Comment:
[OPTIONAL] I also suggest you to make your back button a bit more visible. Also, removing underline from it.

<img width="319" alt="Screen Shot 2022-10-21 at 19 55 11" src="https://user-images.githubusercontent.com/69011963/197226640-7b379fd0-f838-4ac9-843d-b2209c5ac1a1.png">